src/preferences.c

```diff
@@ -26,19 +38,19 @@ void InitializePreferenceFile() {
       realloc(kSerializedJsonWithNull, kSerializedJsonWithNullLength + 1);
   kSerializedJsonWithNewline[kSerializedJsonWithNullLength] = '\n';
 
-  FILE* preferences = fopen("preferences.json", "w");
   fputs(kSerializedJsonWithNewline, preferences);
   fclose(preferences);
   free(kSerializedJsonWithNewline);
+
+  cJSON_Delete(kJsonPreferences);
```

Comment:
@SebastianMygind Ok, I see now what you refer to as a "memory leak". Good addition 👍